<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SplitterHandleWidgetUI</class>
 <widget class="QWidget" name="SplitterHandleWidgetUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>27</width>
    <height>300</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>SplitHCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>73</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QToolBar" name="toolBar">
        <property name="windowTitle">
         <string>edit</string>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
        <addaction name="actionNew_Project"/>
        <addaction name="actionDelete_Project"/>
        <addaction name="separator"/>
        <addaction name="actionNew_Site"/>
        <addaction name="actionDelete_Site"/>
        <addaction name="separator"/>
        <addaction name="actionMove_Up"/>
        <addaction name="actionMove_Down"/>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
  <action name="actionNew_Project">
   <property name="icon">
    <iconset resource="../r.qrc">
     <normaloff>:/img/Resource/137_Add_32.png</normaloff>:/img/Resource/137_Add_32.png</iconset>
   </property>
   <property name="text">
    <string>New Project</string>
   </property>
  </action>
  <action name="actionDelete_Project">
   <property name="icon">
    <iconset resource="../r.qrc">
     <normaloff>:/img/Resource/15501_Remove_32.png</normaloff>:/img/Resource/15501_Remove_32.png</iconset>
   </property>
   <property name="text">
    <string>Delete Project</string>
   </property>
  </action>
  <action name="actionNew_Site">
   <property name="icon">
    <iconset resource="../r.qrc">
     <normaloff>:/img/Resource/13205_SmartArtAddShapeBefore_32.png</normaloff>:/img/Resource/13205_SmartArtAddShapeBefore_32.png</iconset>
   </property>
   <property name="text">
    <string>New Site</string>
   </property>
  </action>
  <action name="actionDelete_Site">
   <property name="icon">
    <iconset resource="../r.qrc">
     <normaloff>:/img/Resource/27785_RemoveClosedCaption_32.png</normaloff>:/img/Resource/27785_RemoveClosedCaption_32.png</iconset>
   </property>
   <property name="text">
    <string>Delete Site</string>
   </property>
  </action>
  <action name="actionMove_Up">
   <property name="icon">
    <iconset resource="../r.qrc">
     <normaloff>:/img/Resource/16803_SmartArtReorderUp_32.png</normaloff>:/img/Resource/16803_SmartArtReorderUp_32.png</iconset>
   </property>
   <property name="text">
    <string>Move Up</string>
   </property>
  </action>
  <action name="actionMove_Down">
   <property name="icon">
    <iconset resource="../r.qrc">
     <normaloff>:/img/Resource/16804_SmartArtReorderDown_32.png</normaloff>:/img/Resource/16804_SmartArtReorderDown_32.png</iconset>
   </property>
   <property name="text">
    <string>Move Down</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../r.qrc"/>
 </resources>
 <connections/>
</ui>
